Automattic/Gravatar-SDK-iOS
# Implementing `UIContentConfiguration` pattern on large profile views

Closes #216 

### Description

Implementing `UIContentConfiguration` pattern on large profile views.

<img src="https://github.com/Automattic/Gravatar-SDK-iOS/assets/9772967/182b0222-8078-4b08-94e9-62c3547b552d" width="40%">

### Testing Steps

- Modify `DemoProfileConfigurationViewController:10` to create a `large` or `largeSummary` style configuration.
- On the demo app -> profile card configuration screen
  - Create add a new profile to the list.
  - Check that the created cell contains the profile view with the desired style.
